### ISSUE TITLE
Failed to compile , typeorm module not found, can't resolve

### ISSUE THREAD

**Issue by AdrianScott** [Globallager/aztec-frontend-boilerplate#4]
getting this error

and that the compile failed "Failed to compile."

./node_modules/typeorm/browser/driver/react-native/ReactNativeDriver.js
Module not found: Can't resolve 'react-native-sqlite-storage' in '/home/adrian/Documents/dev/aztec/aztec-frontend-boilerplate/node_modules/typeorm/browser/driver/react-native'

typeorm is in yarn.lock

grep typeorm *
grep: node_modules: Is a directory
grep: public: Is a directory
grep: src: Is a directory
yarn.lock:    typeorm "0.2.38"
yarn.lock:typeorm@0.2.38:
yarn.lock:  resolved "https://registry.yarnpkg.com/typeorm/-/typeorm-0.2.38.tgz#2af08079919f6ab04cd17017f9faa2c8d5cd566f"

tried yarn add react-native-sqlite-storage 

but that still gives similar error for typeorm and react-native

web console in browser shows:
./node_modules/react-native-sqlite-storage/lib/sqlite.core.js
Module not found: Can't resolve 'react-native' in '... aztec-frontend-boilerplate/node_modules/react-native-sqlite-storage/lib' index.js:1

Uncaught ReferenceError: process is not defined
    4043 blank line 343323 > injectedScript:2
    r blank line 343323 > injectedScript:2
    8048 blank line 343323 > injectedScript:2
    r blank line 343323 > injectedScript:2
    8641 blank line 343323 > injectedScript:2
    r blank line 343323 > injectedScript:2
    <anonymous> blank line 343323 > injectedScript:2
    <anonymous> blank line 343323 > injectedScript:2
    <anonymous> blank line 343323 > injectedScript:2
    onload index.js:1
    be index.js:1
    de index.js:1
    handleErrors webpackHotDevClient.js:169
    onmessage webpackHotDevClient.js:213
    js webpackHotDevClient.js:195
    Webpack 7

**Comment by Globallager** [comment]
Hi, could you share:
1. Which environment (Windows / MacOS / Linux) were you hosting on?
2. What commands did you run in what order?
Thanks!

**Comment by AdrianScott** [comment]
ubuntu

just retried again with fresh version of latest commit and it's working fine

there may be some yarn setup or something i did since trying last night also, not sure... probably can close the issue but might be worth having someone try a very fresh install at some point... thx

**Comment by critesjosh** [comment]
i just ran into this issue with someone. they resolved it by using node version 14.

**Comment by Globallager** [comment]
Thanks guys! @critesjosh did they mention which Node version were they on?

**Comment by critesjosh** [comment]
I think they were on 16. It's strange though because I see other people on M1s using node 16 and they didn't have this error. I'm on an m1 using node 18 no problem

**Comment by AdrianScott** [comment]
In my case I installed v16 LTS after checking for current node install, but then it turned out somehow I had v12 installed from an ubuntu nodejs package, noticed by checking node -v, so i removed that package and perhaps updated .bashrc. This might have been when I was focusing on something else. Then when I came back to working on it again and did a fresh install, it worked. (I had also been experimenting w/ yarn add or npm install typeorm but didn't need to do that in the fresh install)
